contents/index.html

```diff
@@ -12,35 +12,35 @@ <h1 class="big"><span class="main">Ebiten</span><span class="sub">A dead simple
     <p class="img"><img src="images/gopherandgame.png" width="450" alt="Gopher and game developing" style="padding-top: 36px;"></p>
   </div>
 </div>
-<h2 lang="en">What's Ebiten?</h2>
+<h2 lang="en">What is Ebiten?</h2>
 <h2 lang="ja">Ebiten の特徴</h2>
 <div class="grid-container">
   <div class="grid-item-2">
     <h3 lang="en">Dead Simple</h3>
     <h3 lang="ja">とてもシンプル</h3>
     <p class="img"><img src="images/deadsimple.png" width="448" height="224" alt="Dead Simple"></p>
-    <p lang="en">Most rendering operations are represented as drawing an image to an image. Everything is an image: the screen, data from an image file and an offscreen are represented as image objects.</p>
+    <p lang="en">In Ebiten, everything is an image: the screen, data from an image file, and even offscreen items are all represented as image objects. Most rendering operations consist of drawing one image on top of another.</p>
     <p lang="ja">ほとんどの描画命令は画像から画像への描画として表現されます。画面、画像ファイル、オフスクリーンなど、あらゆるものが画像として表現されます。</p>
   </div>
   <div class="grid-item-2">
-    <h3 lang="en">Multi Platforms</h3>
+    <h3 lang="en">Multiplatform</h3>
     <h3 lang="ja">マルチプラットフォーム</h3>
-    <p class="img"><img src="images/multiplatforms.png" width="448" height="224" alt="Multi Platforms"></p>
-    <p lang="en">Ebiten games work on various platforms like desktops (Windows, macOS, Linux, and FreeBSD), web browsers (<a href="https://github.com/gopherjs/gopherjs">GopherJS</a> and WebAssembly), and mobiles (Android and iOS).</p>
+    <p class="img"><img src="images/multiplatforms.png" width="448" height="224" alt="Multiplatform"></p>
+    <p lang="en">Ebiten games work on desktop (Windows, macOS, Linux, and FreeBSD), web browsers (through <a href="https://github.com/gopherjs/gopherjs">GopherJS</a> and WebAssembly), and even on mobile (Android and iOS)! Plus, Ebiten is implemented in pure Go on Windows, so Windows developers do not need to install a C compiler.</p>
```

Comment:
@hajimehoshi -- I added an extra sentence about Windows not needing cgo here. You might want to do the same for the Japanese version on the next line. :+1:

Reply:
Acknowledged.